var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -32,54 +33,154 @@ class UfsWeatherModel(CMakePackage):
         default="FV3_GFS_v15p2,FV3_RRFS_v1alpha",
         description="CCPP suites to compile",
         values=("FV3_GFS_v15p2", "FV3_RRFS_v1alpha", "FV3_GFS_v15p2,FV3_RRFS_v1alpha"),
         multi=True,
+        when="@:2.0.0",
+    )
+    dev_ccpp_default = [
+        "FV3_GFS_v16",
+        "FV3_GFS_v16_flake",
+        "FV3_GFS_v17_p8",
+        "FV3_GFS_v17_p8_rrtmgp",
+        "FV3_GFS_v15_thompson_mynn_lam3km",
+        "FV3_WoFS_v0",
+        "FV3_GFS_v17_p8_mynn",
+        "FV3_GFS_v17_p8_ugwpv1",
+    ]
+    variant(
+        "ccpp_suites",
+        default=",".join(dev_ccpp_default),
+        description="CCPP suites to compile",
+        multi=True,
+        when="@develop",
     )
-    variant("inline_post", default=False, description="Compile post processing inline")
-    variant("multi_gases", default=False, description="Enable multi gases in physics routines")
-    variant("openmp", default=True, description="Enable OpenMP")
-    variant("parallel_netcdf", default=True, description="Enable parallel I/O in netCDF")
     variant(
         "quad_precision",
         default=False,
         description="Enable quad precision for certain grid metric terms in dycore",
+        when="@:2.0.0",
     )
+    variant("mom6solo", default=False, description="Build MOM6 solo executable", when="@develop")
+
     variant(
-        "simdmultiarch", default=False, description="Enable multi-target SIMD instruction sets"
+        "cmake_platform",
+        default="auto",
+        description="Override CMAKE_Platform ('linux.intel', 'hera.gnu', 'acorn', etc.)",
     )
 
+    variant("app", default="ATM", description="UFS application", when="@develop")
+
     depends_on("bacio")
-    depends_on("esmf@:8.0.0")
-    depends_on("mpi")
-    depends_on("nemsio")
+    depends_on("mpi", when="+mpi")
     depends_on("netcdf-c")
     depends_on("netcdf-fortran")
     depends_on("sp")
     depends_on("w3emc")
-    depends_on("w3nco")
-    depends_on("python", type="build")
+    depends_on("esmf@:8.0.0", when="@:2.0.0")
+    depends_on("nemsio", when="@:2.0.0")
+    depends_on("w3nco", when="@:2.0.0")
+    depends_on("bacio@2.4.0:", when="@develop")
+    depends_on("crtm", when="@develop")
+    depends_on("esmf@8.3.0:", when="@develop")
+    depends_on("fms@2022.04: +deprecated_io precision=32,64 constants=GFS", when="@develop")
+    depends_on("g2", when="@develop")
+    depends_on("g2tmpl", when="@develop")
+    depends_on("hdf5+hl+mpi", when="@develop")
+    depends_on("ip@:4", when="@develop")
+    depends_on("netcdf-c~parallel-netcdf+mpi", when="@develop")
+    for app in [
+        "ATMW",
+        "ATML",
+        "NG-GODAS",
+        "S2S",
+        "S2SA",
+        "S2SW",
+        "S2SWA",
+        "S2SWAL",
+        "HAFS",
+        "HAFSW",
+        "HAFS-ALL",
+        "LND",
+    ]:
+        depends_on("parallelio@2.5.3: +fortran~pnetcdf~shared", when="@develop app=%s" % app)
+    depends_on("python@3.6:", when="@develop")
```

Comment:
The default `type` is build/link. Is Python also needed in the `PATH` or `PYTHONPATH` at run-time?

Reply:
No, it's just needed at build time. I just set `type="build"`.